lib/plugins/tag/asset_link.js

```diff
@@ -30,7 +29,7 @@ module.exports = ctx => {
     const attrTitle = escapeHTML(title);
     if (escape === 'true') title = attrTitle;
 
-    const link = encodeURL(resolve(ctx.config.root, asset.path));
+    const link = encodeURL((ctx.config.root + asset.path).replace(/\\/g, '/').replace(/\/{2,}/g, '/'));
```

Comment:
`url_for()` currently cannot be used here because it doesn't convert backward to forward slash.

Reply:
https://github.com/hexojs/hexo-util/pull/228